script/tool/lib/src/create_all_packages_app_command.dart

```diff
@@ -2,49 +2,58 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:io' as io;
-
 import 'package:file/file.dart';
+import 'package:meta/meta.dart';
 import 'package:path/path.dart' as p;
 import 'package:platform/platform.dart';
 import 'package:pub_semver/pub_semver.dart';
 import 'package:pubspec_parse/pubspec_parse.dart';
 
 import 'common/core.dart';
+import 'common/file_utils.dart';
 import 'common/package_command.dart';
 import 'common/process_runner.dart';
 import 'common/repository_package.dart';
 
-const String _outputDirectoryFlag = 'output-dir';
+@visibleForTesting
 
```

Comment:
Whitespace after visibleForTesting

Reply:
Fixed, I forgot the autoformatter wants it after the declaration comment.